docs/openapi.yaml

```diff
@@ -268,9 +268,8 @@ paths:
           required: false
           schema:
             type: integer
-            default: 96
-            maximum: 200
-            example: 100
+            maximum: 50
```

Comment:
Looks like the default is 20. Can we add that field back in?

Reply:
Trying to learn,  "limit": 20, is the limit considered as the default value?

Reply:
Done. Updated.

Reply:
`limit` is the name of the parameter. The default value is what this limit is set to if no parameter is passed.